presto-hive-common/src/test/java/com/facebook/presto/hive/zorder/TestZOrder.java

```diff
@@ -40,10 +40,15 @@
             {42, 43, 46, 47, 58, 59, 62, 63}};
 
     private static final ZValueRange[][] SEARCH_CURVE_RANGES = {
-            {new ZValueRange(Optional.of(4), Optional.of(5)), new ZValueRange(Optional.of(2), Optional.of(3))},
-            {new ZValueRange(Optional.of(3), Optional.of(4)), new ZValueRange(Optional.of(3), Optional.of(4))},
-            {new ZValueRange(Optional.of(4), Optional.of(5)), new ZValueRange(Optional.of(0), Optional.of(3))},
-    };
+            {
+                    new ZValueRange(ImmutableList.of(Optional.of(4)), ImmutableList.of(Optional.of(5))),
+                    new ZValueRange(ImmutableList.of(Optional.of(2)), ImmutableList.of(Optional.of(3)))},
+            {
+                    new ZValueRange(ImmutableList.of(Optional.of(3)), ImmutableList.of(Optional.of(4))),
+                    new ZValueRange(ImmutableList.of(Optional.of(3)), ImmutableList.of(Optional.of(4)))},
+            {
+                    new ZValueRange(ImmutableList.of(Optional.of(4)), ImmutableList.of(Optional.of(5))),
+                    new ZValueRange(ImmutableList.of(Optional.of(0)), ImmutableList.of(Optional.of(3)))}};
```

Comment:
We probably wanna pair up the indentation for each `{` and `}`

Reply:
I've tried pairing the indentations. I'm not sure if you want any different changes, so I'll keep this comment unresolved.